### PR TITLE
Translate hard coded strings in the fees tooltip breakdown

### LINKED ISSUE #8620: Fee types in fees brakedown tooltip are not internationalised
### Describe the bug

In the `payments->settings` screen we have a tooltip for each payment method that displays fees brakedown. In there, we don't internationalise fee types, eg, "base fee", "International payment method fee", and "Foreign exchange fee". 

### To Reproduce

1. Go to Payment->settings
2. Open the fees breakdown tooltip for any payment method
3. Observe the titles
4. Change the language
5. Observe that titles are not internationalised

### PR DESCRIPTION
**Pull request by mgascam**
Fixes #8620 

#### Changes proposed in this Pull Request

This pull request includes changes to the `client/utils/account-fees.tsx` file to wrap static text strings with the localization function to ensure they can be translated.

#### Testing instructions

* Navigate to Payments > Settings and hover the fee "pill" next to any payment method to trigger the fees breakdown tooltip.
* Check the tooltip content is rendered correctly
Screenshot 2024-12-19 at 16 59 36

Translations are not available yet, but you can optionally check that the strings in the scope of this PR are included in the `pot` file. To do so:
1. Build the client so the `pot` file is generated: `npm run build:client`.
2. Check that it contains the new translatable strings:
```shell
grep "International payment method fee" languages/woocommerce-payments.pot
msgid "International payment method fee"
``` 

Or open the `languages/woocommerce-payments.pot` file and manually search for the strings. It should show something like this:
Screenshot 2024-12-19 at 17 07 31

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : 'QA Testing Not Applicable'
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `10004` or branch name `fix/8620-fee-types-in-fees-brakedown-tooltip-are-not-internationalised` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=fix/8620-fee-types-in-fees-brakedown-tooltip-are-not-internationalised) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 1fa47d774a48906f3770bfc982eed3032fbe341c
- Build time: 2024-12-20 09:08:53 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/utils/account-fees.tsx [hunk 1]
```diff
@@ -167,13 +172,21 @@ export const formatMethodFeesTooltip = (
 			{ hasFees( accountFees.fx ) ? (
 				<div>
 					<div>Currency conversion fee</div>
+					<div>
+						{ __(
+							'Currency conversion fee',
+							'woocommerce-payments'
+						) }
+					</div>
```
Comment (htdat) [inline client/utils/account-fees.tsx]: Line 174 should be deleted. That's why the JS snapshot test is failing. This also reflects in the UI: 

Markup on 2024-12-20 at 10:13:08

**Review by htdat — APPROVED** [review]
Overall, LGTM hence pre-approve! But make sure you're deleting the old line as mentioned in my inline comment.

File: client/utils/account-fees.tsx (hunk 1 above)
Comment (mgascam) [reply client/utils/account-fees.tsx]: Oops, I'm afraid I did not resolve conflicts well 😬 Thanks for catching it @htdat !!!

Comment (mgascam) [reply client/utils/account-fees.tsx]: Removed the duplicated string in [1fa47d7](https://github.com/Automattic/woocommerce-payments/pull/10004/commits/1fa47d774a48906f3770bfc982eed3032fbe341c)

Screenshot 2024-12-20 at 10 07 22

**Review by deepakpathania — APPROVED** [review]
LGTM

**Pull request merged**